swagger_to_sdk_config.json

```diff
@@ -6,7 +6,7 @@
   },
   "projects": {
     "azure_mgmt_analysis_services": {
-      "swagger": "arm-analysisservices/2016-05-16/swagger/analysisservices.json",
+      "swagger": "specification/analysisservices/resource-manager/readme.md",
```

Comment:
I like the idea of using the `build_dir` to include the metadata for our reference. @sarangan12 Feel free to open an issue to keep this onto our radar.